teuthology/task/internal/syslog.py

```diff
@@ -107,7 +107,7 @@ def syslog(ctx, config):
             stdout = rem.sh(
                 [
                     'egrep', '--binary-files=text',
-                    '\\bBUG\\b|\\bINFO\\b|\\bDEADLOCK\\b',
+                    '\\bBUG\\b|\\bINFO\\b|\\bDEADLOCK\\b|\\bceph\\b.*\\bWQ_UNBOUND\\b',
```

Comment:
> [Sun Sep 10 07:19:00 2023] workqueue: delayed_work [ceph] hogged CPU for >10000us 4 times, consider switching to WQ_UNBOUND
> [Sun Sep 10 08:41:24 2023] workqueue: ceph_con_workfn [libceph] hogged CPU for >10000us 4 times, consider switching to WQ_UNBOUND

This regex would match only the first message (ceph), but not the second (libceph).

Reply:
> > [Sun Sep 10 07:19:00 2023] workqueue: delayed_work [ceph] hogged CPU for >10000us 4 times, consider switching to WQ_UNBOUND
> > [Sun Sep 10 08:41:24 2023] workqueue: ceph_con_workfn [libceph] hogged CPU for >10000us 4 times, consider switching to WQ_UNBOUND
> 
> This regex would match only the first message (ceph), but not the second (libceph).

Okay, let me run the test again.



> Also, I wouldn't cling to `WQ_UNBOUND` -- it's an implementation detail that could change. If the concern is work items taking longer than expected to run, it would probably be better to match on "workqueue: ... hogged CPU for ...".

I just want to match the ceph ones, this will match all the possible warnings, is that okay ?

Reply:
> I just want to match the ceph ones

That is totally fine -- I just wanted to say that we shouldn't be matching on name of the internal flag.